examples/cloudwatch_listener_ex.py

```diff
@@ -0,0 +1,25 @@
+from locust import main, events
+from locust_plugins.listeners.cloudwatch import CloudwatchAdapter, ServiceContext
+from locust_plugins import missing_extra
+
+try:
+    import boto3
+except ModuleNotFoundError:
+    missing_extra("boto3", "boto3")
+
+
+@events.init.add_listener
+def on_locust_init(environment, **_kwargs):
+    CloudwatchAdapter(_cloudwatch(), environment, _service_context())
+
+
+def _cloudwatch():
```

Comment:
these two methods make the example harder to understand, I think it is better just to inline them.

Reply:
Will do that.